@asutosh936 Jenkins failures seem relevant: 

```
05:02:11 :streams:checkstyleTest[ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk7-scala2.11/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java:22:8: Unused import - java.util.LinkedHashMap. [UnusedImports]
05:02:16 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk7-scala2.11/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java:24:8: Unused import - java.util.Set. [UnusedImports]
05:02:16 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk7-scala2.11/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java:133:46: ',' is not followed by whitespace. [WhitespaceAfter]
05:02:16 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk7-scala2.11/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java:134:46: ',' is not followed by whitespace. [WhitespaceAfter]
05:02:16 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk7-scala2.11/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java:135:46: ',' is not followed by whitespace. [WhitespaceAfter]
05:02:16  FAILED
```

It is because checksytle, as part of the test, reports some non-acceptable coding styles.

--- a/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java
+++ b/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java
@@ -124,10 +124,10 @@ public class TimeWindowTest {
 
     @Test
     public void shouldReturnMatchedWindowsOrderedByTimestamp() {
-        TimeWindows windows = TimeWindows.of(12L).advanceBy(5L);
-        Map<Long, TimeWindow> matched = windows.windowsFor(21L);
+        final TimeWindows windows = TimeWindows.of(12L).advanceBy(5L);
+        final Map<Long, TimeWindow> matched = windows.windowsFor(21L);
 
-        Long[] expected = matched.keySet().toArray(new Long[matched.size()]);
+        final Long[] expected = matched.keySet().toArray(new Long[matched.size()]);
         assertEquals(expected[0].longValue(), 10L);
         assertEquals(expected[1].longValue(), 15L);
         assertEquals(expected[2].longValue(), 20L);

--- a/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java
+++ b/streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java
@@ -19,9 +19,7 @@ package org.apache.kafka.streams.kstream.internals;
 import org.apache.kafka.streams.kstream.TimeWindows;
 import org.junit.Test;
 
-import java.util.LinkedHashMap;
 import java.util.Map;
-import java.util.Set;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
@@ -130,8 +128,8 @@ public class TimeWindowTest {
         Map<Long, TimeWindow> matched = windows.windowsFor(21L);
 
         Long[] expected = matched.keySet().toArray(new Long[matched.size()]);
-        assertEquals(expected[0].longValue(),10L);
-        assertEquals(expected[1].longValue(),15L);
-        assertEquals(expected[2].longValue(),20L);
+        assertEquals(expected[0].longValue(), 10L);
+        assertEquals(expected[1].longValue(), 15L);
+        assertEquals(expected[2].longValue(), 20L);
     }
 }

